src/pages/api/breadcrumbs.ts

```diff
@@ -95,7 +95,7 @@ async function fetchElements(
       },
     ];
   } else if (fieldName === 'campId') {
-    if (fieldValue !== 'standalone') {
+    if (fieldValue !== 'standalone' && fieldValue !== 'inherited') {
```

Comment:
I don't feel strongly about this, but would it maybe be better with `if (typeof fieldValue == 'number') {`?